### ISSUE TITLE
Restore Deprecated Items

### ISSUE THREAD

**Issue by geo-tp** [The-Alpha-Project/alpha-core#1447]
# Restore Deprecated Items

If a item's display was updated after 0.5.3 then the item is in use for 0.5.3

Items that were already deprecated by version 0.5.3 wouldn't typically receive updates, this means that if an item is marked as deprecated but has a display_id higher than the max_display_id for version 0.5.3, it should be considered in use for version 0.5.3.

An item is considered deprecated if it does not appear in any tables such as quests, loot, crafting, vendors, etc.

This list will evolve as we resolve display_id issues attached to unused items, which is why it must be preserved to restore them.

## <b>List of deprecated items concerned</b>

| Entry  | Name                               |
|--------|------------------------------------|
| 97     | Deprecated Dwarven Novice's Robe   |
| 1028   | Deprecated Dented Skullcap         |
| 1072   | Full Moonshine                     |
| 1119   | Bottled Spirits                    |
| 1192   | Deprecated Overseer's Helm         |
| 1279   | Deprecated Soft Leather Hood       |
| 1363   | Deprecated Captain Sander's Eyepatch|
| 1596   | Ghost Hair Thread                  |
| 1940   | Deprecated Skin of Sweet Rum       |
| 2038   | Deprecated Cougar Head Cap         |
| 2045   | Deprecated Cowl of Forlorn Spirits |
| 2071   | Deprecated Mountain Spring Water   |
| 2273   | Guerrilla Armor                    |
| 2275   | Deprecated Sentinel Coif           |
| 2410   | Smoky Torch                        |
| 2586   | Gamemaster's Robe                  |
| 2588   | Deprecated Red Leather Mask        |
| 2625   | Menethil Statuette                 |
| 2918   | Deprecated Coif of Inner Strength  |
| 2945   | (OLD)Medium Throwing Knife         |
| 2993   | Deprecated Inscribed Leather Helm  |
| 2994   | Deprecated Seer's Monocle          |
| 2995   | Deprecated Burnished Chain Coif    |
| 3001   | Deprecated Medivh's Folly          |
| 3015   | Deprecated Chatter's Rock          |
| 3046   | Deprecated Glinting Scale Crown    |
| 3052   | Deprecated Winter Mail Coif        |
| 3059   | Deprecated Forest Leather Helm     |
| 3063   | Deprecated Deepwood Helm           |
| 3068   | Frostweave Cowl                    |
| 3077   | Deprecated Stonecloth Cowl         |
| 3109   | (OLD)Wicked Throwing Dagger        |
| 3128   | (OLD)Medium Throwing Axe           |
| 3136   | (OLD)Heavy Throwing Axe            |
| 3460   | Johaan's Special Drink             |
| 3529   | Black Night Elf Helm               |
| 3536   | Demon Hunter Blindfold             |
| 3675   | Burnt Out Torch                    |
| 3738   | Brewing Rod                        |
| 3773   | Deprecated Murkwood Sap            |
| 3883   | Deprecated Thick Cloth Hat         |
| 3884   | Deprecated Cured Leather Cap       |
| 3885   | Deprecated Scalemail Cap           |
| 3886   | Deprecated Padded Cloth Hat        |
| 3887   | Deprecated Cuirboulli Cap          |
| 3888   | Deprecated Polished Scale Cap      |
| 3952   | Mesh Belt                          |
| 3953   | Mesh Boots                         |
| 3954   | Mesh Bracers                       |
| 3956   | Mesh Gloves                        |
| 3957   | Mesh Pants                         |
| 3958   | Mesh Mantle                        |
| 3959   | Mesh Armor                         |
| 3977   | Strapped Belt                      |
| 3978   | Strapped Boots                     |
| 3979   | Strapped Bracers                   |
| 3981   | Strapped Gloves                    |
| 3982   | Strapped Pants                     |
| 3983   | Strapped Shoulderpads              |
| 3988   | Plate Wall Shield                  |
| 4015   | Sterling Chain Armor               |
| 4193   | Deprecated Feathered Helm          |
| 4441   | MacKreel's Moonshine               |
| 4754   | Deprecated Empty Skin              |
| 4990   | Scorched Bands                     |
| 5053   | Deprecated Plain Brown Robe        |
| 5229   | Empty Greater Bloodstone           |
| 5265   | Watered-down Beer                  |
| 5307   | Deprecated Skipper's Hat           |
| 5333   | Deprecated Mathystra Relic         |
| 5358   | Deprecated Whisperwind Headdress   |
| 5373   | Lucky Charm                        |
| 5625   | Deprecated Band of the Order       |
| 5768   | Gnome Race Ticket                  |
| 5875   | Harness: Blue Ram                  |
| 6064   | Miniature Platinum Discs           |
| 6089   | Zraedus's Brew                     |
| 6173   | Snow Boots                         |
| 6174   | Twain Random Sword                 |
| 6295   | 15 Pound Mud Snapper               |
| 6311   | 22 Pound Catfish                   |
| 6353   | Small Chest                        |
| 6354   | Small Locked Chest                 |
| 6355   | Sturdy Locked Chest                |
| 6356   | Battered Chest                     |
| 6363   | 26 Pound Catfish                   |
| 6364   | 32 Pound Catfish                   |

## SQL Query

```
SELECT i.entry, i.name FROM item_template i
LEFT JOIN creature_loot_template clt ON i.entry = clt.item
LEFT JOIN reference_loot_template rlt ON i.entry = rlt.item
LEFT JOIN quest_template qt ON i.entry IN (
    qt.RewChoiceItemId1, qt.RewChoiceItemId2, qt.RewChoiceItemId3, qt.RewChoiceItemId4,
    qt.RewChoiceItemId5, qt.RewChoiceItemId6,qt.RewItemId1, qt.RewItemId2, qt.RewItemId3, qt.RewItemId4)
LEFT JOIN npc_vendor_template nvt ON i.entry = nvt.item
LEFT JOIN npc_vendor nv ON i.entry = nv.item
LEFT JOIN gameobject_loot_template glt ON i.entry = glt.item
LEFT JOIN playercreateinfo_item pci ON i.entry = pci.itemid
LEFT JOIN alpha_dbc.Spell s ON i.entry IN (s.EffectItemType_1, s.EffectItemType_2, s.EffectItemType_3)
WHERE clt.item IS NULL      -- Not present in creature_loot_template
  AND rlt.item IS NULL      -- Not present in reference_loot_template
  AND qt.entry IS NULL      -- Not referenced in a quest
  AND nvt.item IS NULL      -- Not sold by an NPC template
  AND nv.item IS NULL       -- Not sold by an NPC
  AND glt.item IS NULL      -- Not present in gameobject_loot_template
  AND pci.itemid IS NULL    -- Not given at character creation
  AND s.ID IS NULL          -- Not present as an item craft in a spell effect
  AND i.entry < 6439        -- Filter for entry, less than max item entry for 0.5.3
  AND i.display_id > 11802; -- Filter for display_id, greater than max item display_id for 0.5.3
```

**Comment by VictoriumWoW** [comment]
Did some research into this subject. Here are my findings:

**DEPRECATED NPC DROPS**
These are items that have a naming convention that corresponds to unique loot from an NPC, implying they were intended to be dropped from NPCs but for whatever reason were cut. The following list contains every item (excluding the Deprecated tag), their ID, and which NPC was likely meant to drop them.
1.	Overseer's Helm (1192) - Dropped from Riverpaw Overseer (125)
2.	Guerrilla Armor (2273) - Dropped from Dark Iron Guerilla (1171)
3.	Mark of the Kirin Tor (5004) - Dropped from Dalaran Summoner (2358)
4.	Whisperwind Headdress (5358) - Dropped from Earthcaller Halmgar (4842)
5.	Band of the Order (5625) - Dropped from Stromgarde Vindicator (2585)


**DEPRECATED QUEST REWARDS**
These are items that have a naming convention that corresponds to quest rewards, implying they were intended to be awarded from quests but for whatever reason were cut. The following list contains every item (excluding the Deprecated tag), their ID, and which quest they likely were meant to be a reward from.
1.	Soft Leather Hood (1279) - Awarded from Protecting the Herd (314)
2.	Captain Sander's Eyepatch (1363) - Awarded from Captain Sander's Hidden Treasure (140)
3.	Aegis of Westfall (2040) - Awarded from The Defias Brotherhood (166)
4.	Cowl of Forlorn Spirits (2045) - Awarded from The Legend of Stalvan (98)
5.	Coif of Inner Strength (2918) - Awarded from A Grim Task (304)
6.	Skipper's Hat (5307) - Awarded from The Guns of Northwatch (891)
7.	Deckhand Gloves (5308) - Awarded from The Guns of Northwatch (891)

**DEPRECATED SET ITEMS**
These are items that have a naming convention that corresponds to a world drop set, implying they were intended to be apart of sets but for whatever reason were cut. The following list contains every item (excluding the Deprecated tag), their ID, and which set they're apart of.
1.	Seer's Monocle (2071) - Apart of the Seer's set
2.	Inscribed Leather Helm (2993) - Apart of the Inscribed Leather set
3.	Burnished Chain Coif (2995) - Apart of the Burnished Chain set
4.	Glinting Scale Crown (3046) - Apart of the Glinting Scale set
5.	Forest Leather Helm (3059) - Apart of the Forest Leather set
6.	Frostweave Cowl (3068) - Apart of the Frostweave set
7.	Thick Cloth Hat (3883) - Apart of the Thick Cloth set
8.	Cured Leather Cap (3884) - Apart of the Cured Leather set
9.	Scalemail Cap (3885) - Apart of the Scalemail set
10.	Padded Cloth Hat (3886) - Apart of the Padded Cloth set
11.	Cuirboulli Cap (3887) - Appart of the Cuirboulli set
12.	Polished Scale Cap (3888) - Apart of the Polished Scale set
13.	Feathered Helm (4193) - Apart of the Feathered set

**MISC FINDINGDS**
1.	Scorched Bands (4990) seems to have been a reward from an unused quest, or one that has few/no rewards on release. Has a ID right next to Mage Dragon Robes (4989) and Burning Obsidian Band (4988), and all these items have an identical item level. They're also close to the ID of the Dwarf Captain Sword (4987), whose questline exists in 0.5.3, meaning these three items were likely a reward from a level 50+ quest that existed in 0.5.3. Needs research.
2.	Sentinel Coif (2275), Moon Glaive (3933), and the Warden Glaive (3934) may have been apart of the scrapped Hero Class system, in which characters level 40+ could become a more powerful version of their class based on their race (night elf warriors became demon hunters, orc warriors became blademasters, etc.). These items, among others, may be relics of this old system that were never implemented.

**Comment by geo-tp** [comment]
Good research work, we could use these findings to restore them. Do you have any links that helped you find this ?

By using the logic of display_id number again, we could also assume that all deprecated items with a very high display_id (near the max_item_display_id for 0.5.3) are also most likely in use.

I'm not sure from which point we could consider this reliable, but it could eventually apply to all deprecated items with a display_id higher than 11k.

## Deprecated items, display_id 11000 to 11802 : 

|entry|name|display_id|
|-----|----|----------|
|6437|Flayed Demon Skin (old)|11780|
|6435|Infused Burning Gem|11766|
|4385|Green Tinted Goggles|11663|
|4373|Shadow Goggles|11644|
|5782|Thick Murloc Armor|11598|
|4319|Azure Silk Gloves|11546|
|6385|Stylish Green Shirt|11519|
|6384|Stylish Blue Shirt|11518|
|5095|Rainbow Fin Albacore|11483|
|6371|Fire Oil|11461|
|6370|Blackmouth Oil|11459|
|6360|Steelscale Crushfish|11453|
|6359|Firefin Snapper|11451|
|6358|Oily Blackmouth|11450|
|6222|Formula: Imbue Chest - Minor Spirit|11431|
|6343|Formula: Imbue Chest - Spirit|11431|
|6345|Formula: Imbue Cloak - Protection|11431|
|6376|Formula: Enchant Boots - Minor Stamina|11431|
|6339|Runed Silver Rod|11353|
|6338|Silver Rod|11351|
|6316|Loch Frenzy Delight|11268|
|6307|Message in a Bottle|11224|
|6299|Sickly Looking Fish|11210|
|6292|10 Pound Mud Snapper|11201|
|6294|12 Pound Mud Snapper|11201|
|6287|Atal'ai Tablet Fragment|11183|
|6263|Blue Overalls|11182|
|6284|Runes of Summoning|11180|
|3259|Ruined Bat Hide|11164|
|6278|Musty Scroll|11161|
|6277|Musty Parchment|11160|
|2572|Red Linen Robe|11136|
|4344|Brown Linen Shirt|11135|
|4331|Phoenix Gloves|11041|

## Deprecated items, display_id 10 000 to 11 000: 

|entry|name|display_id|
|-----|----|----------|
|6253|Leftwitch's Package|10924|
|6244|ggggfg|10913|
|2585|Gray Woolen Robe|10896|
|6243|Green Woolen Robe|10894|
|2587|Gray Woolen Shirt|10892|
|2578|Barbaric Linen Vest|10891|
|2579|Green Linen Shirt|10861|
|6239|Red Linen Vest|10860|
|6242|Blue Linen Robe|10859|
|2569|Linen Boots|10854|
|2582|Green Woolen Vest|10849|
|2577|Blue Linen Shirt|10845|
|6240|Blue Linen Vest|10843|
|6238|Brown Linen Robe|10841|
|2576|White Linen Shirt|10834|
|6309|17 Pound Catfish|10814|
|6310|19 Pound Catfish|10814|
|5546|Fast Test Crossbow|10673|
|4899|Test Crossbow|10672|
|6219|Arclight Spanner|10657|
|6218|Runed Copper Rod|10656|
|6214|Heavy Copper Maul|10642|
|6193|Bundle of Atal'ai Artifacts|10481|
|6090|OLDNoboru's Cudgel|10453|
|6185|Bear Shawl|10425|
|6150|A Frayed Knot|10301|
|3710|Rod of Helcular|10275|
|6131|Trapper's Pants|10244|
|6130|Trapper's Shirt|10076|
|5739|Barbaric Harness|10018|

## Full list of deprecated items

[deprecated_items_lower_than_displa_id_11802.csv](https://github.com/user-attachments/files/17318423/deprecated_items_lower_than_displa_id_11802.csv)

**Comment by VictoriumWoW** [comment]
I don't have a whole lot of time as of this post, but it's interesting that you brought up the Runes of Summoning. 

This item is actually in-use on release, as apart of the undead warlock quest to learn the Summon Voidwalker spell titled "The Binding" (1471). Turns out, not only do the other quest items that are apart of this chain exist in-game in the 0.5.3 database according to what we know was the last item, but all of the quests in this chain are in the 1471-1478 ID range, which is too high for quests in 0.5.3. Not only that, but this same rule applies for the Summon Imp quest for undead warlocks called "Piercing the Veil" (1470) with its quest items called "Rattlecage Skull" (6281), as well as the Succubus questline with the hearts (Dalin's Heart and Comar's Heart, IDs 6312 and 6313 respectively). We even have the spells in the 0.5.3 database that summon the enemy voidwalker/succubus at the summoning circles, which provide a "Requires Summoning Circle" error message when you try to use them in-game (https://db.thealphaproject.eu/index.php?action=show_spell&id=7728&filter=summon+voidwalker&sort_order=Name_enUS&pos=3&max=3, https://db.thealphaproject.eu/index.php?action=show_spell&id=7729&filter=summon+succubus&sort_order=Name_enUS&pos=3&max=3). Also, the projections of Dalin and Comar which briefly spawn during a cutscene in said succubus questline exist in 0.5.3 with their proper displayIDs (https://db.thealphaproject.eu/index.php?action=show_creature&id=5691&sort_order=entry&page=25&pos=42&max=4909, https://db.thealphaproject.eu/index.php?action=show_creature&id=5692&sort_order=entry&page=25&pos=43&max=4909).

With all of this evidence, it seems like what we believed to be the last quest in 0.5.3 (1339) is actually wrong, and the final quest was much higher. I can't say if these quests were available to players, but given this evidence it's very believable that these quests were at least late into development during 0.5.3 (which would mean they would exist in the database but with the <NYI> tag and no way to start them,) and because of their high IDs, every quest underneath them was either also in development or was available to players (should be handled in a case-by-case basis.) This would fix issues like the missing quest in The Missing Diplomat, making it completable. 

TLDR: The last quest in 0.5.3 may not be 1339. Until we find a definitive last quest, every quest that is at or below ID 1478 should be reimplemented.

**Comment by devw4r** [comment]
Regarding warlock quests, some other game data also suggest that the quests could've existed.

QuestSort.dbc has as final ID the 'Warlock' type quest.
```
1:Epic
23:Undercity
21:Wailing Caverns
22:Blackfathom Deeps
24:Ironforge
25:Monastery
41:Uldaman
61:Warlock
```

SpellFocusObject.dbc has as final ID `43:Summoning Circle`

**Comment by geo-tp** [comment]
I don't know how the max quest was determined, but 1339 seems consistent. The quests following involve items or NPC higher than max entries

There are several gaps between the last quest entry, in some cases when blizz update a quest, they delete the old one and create a new entry

This could perhaps be related to the warlock quests and explain why items are ingame but the reworked quests are not


see gaps between 1324-1338 and 1339-1358
|entry|Title|Details|
|-----|-----|-------|
|1323|The Black Shield|Oh, I think I might have dropped too much acid onto the shield. Let me just rub a little...$b$bHm, kind of hard to read, but it looks like the name Mosarn. Never heard of him. Maybe Krog knows. Krog knows many people.|
|1324|The Missing Diplomat|What?! You come here accusing me of allying with the Defias, and think I'll just stand here while you insult me?! Something tells me you've had this coming for a long time.|
|1338|Stormpike's Order|There is a dwarven armorsmith whose work I admire greatly.  His name is Furen Longbeard, and his skill is unmatched.  I need a new shield, and I must have one of his!$B$BThe problem is... Furen is far to the south, in the dwarven district within Stormwind.  In human lands!  I can't imagine why he would live there, so far from Ironforge.  The humans must pay him a king's ransom to keep him there!$B$BSo, if you want to travel and don't mind making some money, then will you send Furen my request for a shield?|
|**>1339<**|**Mountaineer Stormpike's Task**|**I hear Mountaineer Stormpike is looking for a runner.  Someone to do a little traveling for him.  How about it?  Are you the  for the job?$B$BIf so, then you'll find Stormpike at the top of the northern guard tower.**|
|1358|Sample for Helbrim|A colleague in Kalimdor, Apothecary Helbrim, is researching the effects of Lordaeron toxins on the beasts of Kalimdor.  He would be very interested in a sample of the wolf hearts Renferrel sent me.$B$BHelbrim is stationed in the Barrens, in a village known as the Crossroads.  To reach him, you must travel via zeppelin to the orc capital of Orgrimmar, then travel south, by foot, to Durotar.  Head west into the Barrens, and you will soon reach the Crossroads.|
|1359|Zinge's Delivery|The wolf hearts you acquired possess a quality unseen in similar specimens.  There is a toxicity to them that should be deadly to the very wolves in which the hearts beat!$B$BIt's quite incredible, and we must study it further.  I kept a few samples of the hearts acquired.  Take them to my colleague, Apothecary Zinge.  You will find her in our headquarters in the Undercity.|
|1360|Reclaimed Treasures|I had myself a fine job as a digger at the Uldaman excavation site in the Badlands... that is until those blasted troggs showed up.  I escaped with my hide barely intact, and nothing else.  In the chaos, I left my most treasured possession.  It's safe from the troggs in my chest, sure, but it's right in the thick of 'em.$B$BI'd be in your debt, both in honor and coin, if you could fetch it for me!  A chest with my name on it is at the end of the North Common Hall.  That's the one you need to get in.|
|1361|Regthar Deathgate|A warrior of the Horde, Regthar Deathgate, is recruiting agents for a mission into the land of Desolace.  Speak with him if you wish to volunteer.  He is stationed in the Barrens, in a series of bunkers west of the Crossroads.$B$BAnd sharpen your weapons.  Desolace is a place torn by war.|
|1362|The Kolkar of Desolace|The Kolkar centaur live in Desolace, and their forces have spread into the Barrens.  This is a known fact and we have Horde agents dealing with them here.$B$BBut as long as the centaur are strong in Desolace, they will always pose a threat.  It is time we rid ourselves of that threat.$B$BSpeak with Felgur Twocuts at Ghost Walker Post in Desolace.  He was sent to gather intelligence on the centaur.  He will know how to best deal with the Kolkar of Desolace.|
|1363|Mazen's Behest|As caretaker of the works here in the library of the Academy of Arcane Arts and Sciences, it is my responsibility to ensure all the magical tomes and scrolls are accounted for at all times. Who knows what folly would unfold if some of these enchanted texts were to fall into the wrong hands?$b$bWhich reminds me... my assistant, Dellis, made a horrible mistake last week. He's quite able in the library but incompetent when he leaves the gates of Stormwind. Be a hero, and help old Dellis out, would you?|
|1364|Mazen's Behest|Master Mazen's directions were quite clear: Deliver a very important essay to Watcher Mahar Ba in Nethergarde Keep.$b$bBut I had never been to the Blasted Lands before! The journey was frightening!$b$bWhen I got to the Swamp of Sorrows, I made a wrong turn and ended up surrounded by horrible plant-like swamp creatures. They lurched as if to attack me so I ran. But in my haste I dropped the essay. One of the beasts took it I am sure.$b$bCould you please try to retrieve it and deliver it to Mahar Ba?|
|1365|Khan Dez'hepah|The Kolkar are not the only centaur of Desolace.  Many clans wander these dry plains, fighting and dying against each other.  And it is good that they fight among themselves, for they are strong and brutal.  If the centaur united against us, they would be a fierce foe.$B$BWe must deal with them before they unite.$B$BYour first task is to disrupt the Kolkar clan.  Do this by killing its leader, Khan Dez'hepah.$B$BThe Kolkar Village is northeast of here.  Go, and may fortune go with you.|

**Comment by VictoriumWoW** [comment]
With Wayback Machine down, it's difficult for me to find evidence right now of what quests may have been in these quest gaps. Considering the Warlock quest evidence, they are possibly in these gaps alongside other quests like the Flayed Demon Hide questline and the missing quest in The Missing Diplomat. We may have to "adlib" a bit, though, in terms of the IDs.

**Comment by geo-tp** [comment]
Using the same logic, we can assume that all deprecated items with a spell_id higher than the max for 0.5.3 should also be used for 0.5.3

### Deprecated Item with spellID higher than 0.5.3
|entry|name|
|-----|----|
|3687|Deprecated Unholy Avenger|
|5513|Mana Jade|
|6284|Runes of Summoning|
|6286|Pure Hearts|
|6436|Burning Gem|


Same for disenchant_id, if a deprecated item has a disenchant_id, it was still in use after 0.5.3


### Deprecated Item with disenchant_id != 0
|entry|name|
|-----|----|
|2267|Stonesplinter Mace|
|3567|Dwarven Fishing Pole|
|4110|Master Hunter's Bow|
|4111|Master Hunter's Rifle|
|4116|Olmann Sewar|
|5005|Emberspark Pendant|
|5250|Charred Wand|
|6360|Steelscale Crushfish|

**Comment by ratkosrb** [comment]
The disenchant id is a custom column, its not something sent by the server, this shouldn't be used as indication for anything.

**Comment by geo-tp** [comment]
> The disenchant id is a custom column, its not something sent by the server, this shouldn't be used as indication for anything.

I've updated it, thanks for the info.

**Comment by VictoriumWoW** [comment]
Can confirm that Runes of Summoning existed since we have the Summoning Circle-activated Summon Voidwalker spell ID in 0.5.3. Unholy Avenger has a very low ID so it was likely a GM test item, we can also see players with this sword in very old trailers.

**Comment by VictoriumWoW** [comment]
So after some research, it seems like players have access to head items *with stats* at level 20+, with a minimum item level of 30. This is when excluding engineering, of course. This means that all items that have an item level below 30 and/or a level requirement below 20 don't need stats added to them as part of restoration. An example of this would be Captain Sander's Eyepatch, all we'd have to do is change the quality to white, add a level requirement of 10 or 5, give it an existing and proper displayID, and we could reasonably re-add it as-is. However, some items like Band of the Order that do have an item level below 30 may have had a higher item level before being deprecated (in this case, Band of the Order is a sister item [entry ID right next to] to Circlet of the Order, which has an item level of 38, implying that the Band of the Order may have had a higher ilvl.) We'd have to go through each item individually and determine if it has a sister item, is part of a set, and if it does compare the item levels and make the necessary changes.

**Comment by VictoriumWoW** [comment]
There's a few items that I left out of my second Deprecated Item Restoration pull request due to the belief that they may have had stats, similar to Whisperwind Headdress, before their deprecation. The items I flagged as possibly having stats, and thereby left out, are as follows:

1. Orcslayer (believed to be the original reward from Tharil'zun before the rewards from Shadow Magic were moved to it)
2. Wristguards of the Fen Warden (believed to be a reward from Fire Taboo)
3. Cowl of Forlorn Spirits (was certainly a reward from The Legend of Stalavan)

I'd also like to make mention of Militia Handaxe, which was a reward from Brotherhood of Thieves and an item that I did include. Because our only source of this item's damage values comes from 1.12, it has abnormally low damage values compared to the other Militia weapons. Because of this, this weapon and the others like Orcslayer may need rescaling.

**Comment by VictoriumWoW** [comment]
Having done some reseach, I found some evidence that could confirm or debunk some of these items:

![Image](https://github.com/user-attachments/assets/7a541a24-8f13-484d-a564-048390539406)

Excluding the players who are wearing what seems to be Frostweave Cowl, I noticed one player named Crossbone wearing a brown helmet that appears to be the Forest Leather Helm, although this could also be the Enduring Cap which shares its model. However, a lot of players aren't wearing headgear at all. This has led me to believe that the set items (Seer's Monocle, etc.) may have been removed by the time 0.5.3 launched, although it could be that they were just rare.

I found another period screenshot showing the rewards of the quest The Coastal Menace:

![Image](https://github.com/user-attachments/assets/529d16e9-8ea0-4162-9f19-0ac5928ab3e4)

At first, nothing is unordinary, but if we investigate we'll learn that there's a deprecated item called [Mantle of the Seas](https://db.thealphaproject.eu/index.php?action=show_item&id=1545&filter=deprecated&sort_order=name&pos=187&max=357) which has a similar naming convention to the other reward, Buckler of the Seas. That, coupled with the item being very close to the Buckler ID-wise, leads me to believe that it was an original reward from the quest. However, it isn't in the rewards for this quest. Since this item's displayID exists in 0.5.3, this leads me to believe that items with displayIDs that exist in 0.5.3 were deprecated by the time this patch hit. This would include (but is not limited to):

- Mantle of the Seas (obligatory)
- Orcslayer
- Night Mage Wristguards

Items that aren't poor quality should be handled case-by-case, though. For example, the hands of the moon could be exempt from this as they're quest items, which means they're likely related to their questline. The questline may have originally been about upgrading them, which is why you got so many pairs. Also, the Lightforge weapons had to have existed (and currently do exist in the server), as Morbent's Bane didn't exist.
